entrypoint.sh

```diff
@@ -82,6 +82,11 @@ validate_args() {
   fi
 }
 
+lets_wait() {
+  echo "Sleeping for ${wait_interval} seconds"
+  sleep $wait_interval
```

Comment:
This needs a fix I think. Compare it to line 127

Reply:
what exactly does need to be changed? code is in two sections, one has quotes, other does not, both work the same.